price-feeder/oracle/provider/coinbase.go

```diff
@@ -342,25 +342,30 @@ func (p *CoinbaseProvider) messageReceived(messageType int, bz []byte) {
 
 	var coinbaseTrade CoinbaseTradeResponse
 	if err := json.Unmarshal(bz, &coinbaseTrade); err != nil {
-		p.logger.Debug().Msg("unable to unmarshal response")
+		p.logger.Error().Err(err).Msg("unable to unmarshal response")
 		return
 	}
+
 	if coinbaseTrade.Type == "error" {
 		var coinbaseErr CoinbaseErrResponse
 		if err := json.Unmarshal(bz, &coinbaseErr); err != nil {
-			p.logger.Debug().Msg("unable to unmarshal error response")
+			p.logger.Error().Err(err).Msg("unable to unmarshal error response")
```

Comment:
This should probably be a `Debug`

Reply:
done at https://github.com/umee-network/umee/pull/684/commits/15b4b4592c1cff53b3fa7785345ec2a32b86c4fe